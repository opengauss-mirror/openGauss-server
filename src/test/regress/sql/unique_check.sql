create schema unique_check;
set current_schema to 'unique_check';

drop table if exists warehouse;

create table warehouse (
w_id smallint not null,
w_name varchar(10), 
primary key (w_id) );

insert into warehouse values (1, 1);
insert into warehouse values (1, 1);
SET UNIQUE_CHECKS=0;
insert into warehouse values (1, 1);

drop table warehouse;

reset UNIQUE_CHECKS;
drop schema unique_check cascade;
reset current_schema;
