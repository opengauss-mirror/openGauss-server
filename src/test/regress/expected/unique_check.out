create schema unique_check;
set current_schema to 'unique_check';
drop table if exists warehouse;
NOTICE:  table "warehouse" does not exist, skipping
create table warehouse (
w_id smallint not null,
w_name varchar(10), 
primary key (w_id) );
NOTICE:  CREATE TABLE / PRIMARY KEY will create implicit index "warehouse_pkey" for table "warehouse"
insert into warehouse values (1, 1);
insert into warehouse values (1, 1);
ERROR:  duplicate key value violates unique constraint "warehouse_pkey"
DETAIL:  Key (w_id)=(1) already exists.
SET UNIQUE_CHECKS=0;
insert into warehouse values (1, 1);
drop table warehouse;
reset UNIQUE_CHECKS;
drop schema unique_check cascade;
reset current_schema;
